src/lib/game.ts

```
@@ -495,20 +495,24 @@ function getLocalizedWitnesses(LL: TranslationFunctions, place: Place): Localize
 			possibleWitnesses = [Witness.BARTENDER, Witness.TENNIS_PRO, Witness.WAITER];
 			break;
 		case Place.STOCK_EXCHANGE:
-		default:
 			possibleWitnesses = [Witness.ANALYST, Witness.MESSENGER, Witness.TRADER];
+			break;
+		default:
+			possibleWitnesses = [];
 	}
 
+	if (possibleWitnesses.length === 0) throw new Error('No witnesses found for this place.');
```

This error is not surfaced to the UI but might help debug this in the future.